2 - Triplet sampling (Random).ipynb

```diff
@@ -202,22 +210,27 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def random_triplet_sampling(samples_per_user, hashes_container, desc=None):\n",
+    "def random_triplet_sampling(samples_per_user, hashes_container, desc=None, limit_iteration=10000):\n",
     "    interactions = interactions_df.copy()\n",
     "    samples = []\n",
     "    for ui, group in tqdm(interactions.groupby(\"user_id\"), desc=desc):\n",
     "        # Get profile artworks\n",
     "        full_profile = np.hstack(group[\"item_id\"].values).tolist()\n",
     "        full_profile_set = set(full_profile)\n",
     "        n = samples_per_user\n",
+    "        aux_limit = limit_iteration\n",
     "        while n > 0:\n",
+    "            if aux_limit == 0:\n",
+    "                break\n",
```

Comment:
`aux_limit` does not change its value, in line 247 we should use `aux_limit` instead of `limit_iteration` and that may be a fix